mbed_build/_internal/config/config_modifiers/utils.py

```diff
@@ -0,0 +1,26 @@
+#
+# Copyright (C) 2020 Arm Mbed. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+"""Utilities for building config modifiers."""
```

Comment:
Could have a more descriptive name for this file? `utils` is a bit ambiguous tbh. How about `build_modifiers`?

Reply:
https://github.com/ARMmbed/mbed-build/pull/20/commits/f0b8788d13b0ade5e0c2637ca6ca1c8c1d541227